Src/Sarif.PatternMatcher/Fingerprint.cs

```diff
@@ -127,6 +131,11 @@ public override string ToString()
                 components.Add($"[{AccountKeyName}={this.Account.Trim()}]");
             }
 
+            if (!string.IsNullOrEmpty(Database))
```

Comment:
>Database [](start = 38, length = 8)

I have wondered previously whether we should call this 'Resource' or something. in other words, this qualifier is for some host- or server-resident thing.

thoughts?